pkg/run/git_exec.go

```diff
@@ -0,0 +1,44 @@
+package run
+
+import (
+	"os/exec"
+	"strings"
+)
+
+func GetBranchName() (string, error) {
+	cmd := exec.Command("git", "rev-parse", "--abbrev-ref", "HEAD")
+	out, err := cmd.Output()
+
+	if err != nil {
+		return "", err
+	}
+
+	return strings.TrimSpace(string(out)), nil
```

Comment:
❔ Why is trimming the space required here? Can this command return a string with trailing or leading spaces as the branch name?

Reply:
It's for eliminating "\n" "\r" printed out by the command.